templates/settingsForm.tpl

```diff
@@ -0,0 +1,26 @@
+<script>
+	$(function() {ldelim}
+		// Attach the form handler.
+		$('#plagiarismSettingsForm').pkpHandler('$.pkp.controllers.form.AjaxFormHandler');
+	{rdelim});
+</script>
+
+<form class="pkp_form" id="plagiarismSettingsForm" method="post" action="{url router=$smarty.const.ROUTE_COMPONENT op="manage" category="generic" plugin=$pluginName verb="settings" save=true}">
+	{csrf}
+	{include file="controllers/notification/inPlaceNotification.tpl" notificationId="plagiarismSettingsFormNotification"}
+
+	<div id="description">{translate key="plugins.generic.plagiarism.manager.settings.description"}</div>
+	{if $ithenticate_forced}
+	<div id="ithenticate_notice">{translate key="plugins.generic.plagiarism.manager.settings.areForced"}</div>
+	{/if}
+
+	{fbvFormArea id="webFeedSettingsFormArea"}
+            {fbvElement type="text" id="ithenticate_user" value=$ithenticate_user label="plugins.generic.plagiarism.manager.settings.username"}
```

Comment:
NBD, but we generally use `lowerCamelCase` for PHP variables, HTTP parameters, etc. (rather than `underscore_separated_words`).

Reply:
Done.